clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignor.java

```diff
@@ -163,6 +183,7 @@ private boolean allSubscriptionsEqual(Set<String> allTopics,
         Map<String, List<TopicPartition>> assignment = new HashMap<>(
             consumerToOwnedPartitions.keySet().stream().collect(Collectors.toMap(c -> c, c -> new ArrayList<>(minQuota))));
 
+        // step 1: Reassign as many previously owned partitions as possible
```

Comment:
nit: I don't think we need steps here, which makes it hard to squeeze in more comments later.

Reply:
Updated. Thanks @abbccdda .